Update/ura_005.txt

```diff
@@ -1374,7 +1374,7 @@ void main()
 //r迷宮入りの事件で手柄を立てようとするよりも、手近なデータ収集で確実に点数を稼ぐ…学者肌の小才子にはよくある保身だ。
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "迷宮入りの事件で手柄を立てようとするよりも、手近なデータ収集で確実に点数を稼ぐ…学者肌の小才子にはよくある保身だ。",
-			NULL, "Instead of trying to solve an unsolvable case, earn credit by collecting data that can be easily and reliably gathered... that seemed the more prudent means of self-preservation for a thin-skinned bookworm.", Line_Normal);
+			NULL, "Instead of trying to take credit for an unsolvable case, he would earn points by collecting data that can be easily and reliably gathered... That seemed the more prudent means of self-preservation for a thin-skinned bookworm.", Line_Normal);
```

Comment:
```suggestion
			NULL, "Instead of trying to gain credit by solving an unsolvable case, he would earn points by collecting data that can be easily and reliably gathered... That seemed the more prudent means of self-preservation for a thin-skinned bookworm.", Line_Normal);
```